build/generate.mk

```diff
@@ -52,7 +52,7 @@ generate-node-installed:
 .PHONY: generate-autorest-installed
 generate-autorest-installed:
 	@echo "$(ARROW) Detecting autorest..."
-	@which autorest > /dev/null || { echo "run 'npm install -g autorest' to install autorest"; exit 1; }
+	@which autorest > /dev/null || { echo "run 'npm install -g autorest@3.7.2' to install autorest"; exit 1; }
```

Comment:
This locking is great!

Reply:
Should we also check for the installed version and say something like "You don't have 3.7.2 installed"?